src/app/routes/liveRadio/getInitialData/index.test.js

```diff
@@ -20,7 +20,11 @@ describe('Get initial data for live radio', () => {
     expect(pageData.summary).toEqual(
       '세계와 한반도 뉴스를 공정하고 객관적으로 전달해 드립니다',
     );
-    expect(pageData.content.blocks.length).toBeTruthy();
+    expect(pageData.heading).toEqual('BBC 코리아 라디오');
+    expect(pageData.bodySummary).toEqual(
```

Comment:
Summary & body summary output look identical here - do we need both?

Reply:
`promo.summary` and `content.blocks[1].text` are not guaranteed to always have the same values (although I have yet to see them vary)

Reply:
OK thanks for the clarification. I think I would still like to have 1 summary and 1... _something else_ (not bodySummary as this is confusing). See https://github.com/bbc/simorgh/pull/6606#discussion_r431924530

Reply:
For folk not privee to the Slack chat - we've made the decision to tackle the duplication of the same data coming from different sources in a future swarm